server/index.js

```diff
@@ -5,11 +5,12 @@ const morgan = require('morgan')
 const mongoose = require('mongoose')
 const {
   registerListenWatcher,
-  registerStopListeningWatcher
+  registerStopListeningWatcher,
+  web3
```

Comment:
is this being used?

Reply:
I was just moving it to one require statement, didn't add or remove anything